helm/cluster-gcp/templates/_machine_deployments.tpl

```diff
@@ -53,6 +53,7 @@ spec:
       image: {{ include "vmImage" $global }}
       instanceType: {{ .instanceType | default "n2-standard-4" }}
       rootDeviceSize: {{ .rootVolumeSizeGB | default 100 }}
+      serviceAccounts:
```

Comment:
something is missing here

Reply:
I forgot to add those changes. Fixed now.